CMakeLists.txt

```diff
@@ -183,6 +183,8 @@ function (config_with_llvm)
   protobuf_generate_cpp(PERF_PARSER_OPTIONS_CC PERF_PARSER_OPTIONS_HDR third_party/perf_data_converter/src/quipper/perf_parser_options.proto)
   protobuf_generate_cpp(PERF_STAT_CC PERF_STAT_HDR third_party/perf_data_converter/src/quipper/perf_stat.proto)
   add_library(perf_data_proto OBJECT ${PERF_DATA_PROTO_CC})
+  add_library(perf_parser_options OBJECT ${PERF_PARSER_OPTIONS_CC})
+  add_library(perf_stat OBJECT ${PERF_STAT_CC})
```

Comment:
Could you rename them to perf_parser_options_proto and perf_stat_proto for consistency with config_without_llvm?

Reply:
Sure, I pushed a change with the requested rename.